executive board page

minjlys opened the pull request:
<img width="1248" height="825" alt="Screenshot 2025-10-29 at 10 21 11 PM" src="https://github.com/user-attachments/assets/ceb028ef-f9b0-4cd6-a3de-cf7bec2cb370" />

had a few questions + need additional guidance on what to do next

1. the card, role, and name color are pulled from the figma file (used bg-[hexcode] instead of bg-pse-colorname-#) 
- i did not see these colors in globals.css, so put the hexcodes in instead as placeholders

2. i tinified the jpgs of the photos, but they seem blurry to me - should i just leave it like that?

3. probably need to fix the padding because the very bottom card hits the footer

minjlys commented:
merge conflicts my biggest enemy

swedenagu reviewed (changes requested):
Looks good! I'll push a commit for the updated colors

brandontran1220 commented on src/app/page.tsx:
```diff
@@ -1,14 +1,19 @@
       <OurMission />
       <Principles />
-    </>
+    </div>
```
```suggestion
    <>
<OurMission />
<Principles />
    </>
```

brandontran1220 reviewed (changes requested):
Overall, looks pretty good! Just fix up a couple things and we're good to go!

minjlys commented:
**updates:** 
BoardGrid.tsx
- Added pb-10 to fix padding issue 

BoardCard.tsx 
- Swapped LinkedIn.png w/react icon 
- Swapped hex code colors w/colors in globals.css for now - colors do not match to the figma file
- Add w-96 for wider cards

page.tsx 
- Added gap-16, pb-10 

im very sorry about the amount of merge conflicts i get :( i will try to make it a habit to update my local branch BEFORE i work on it

swedenagu commented on src/components/board/BoardCard.tsx:
```diff
@@ -0,0 +1,67 @@
+          height={40}
+          alt="LinkedIn"
+        />
+      </a>
```
added
